karspexet/templates/base.html

```diff
@@ -0,0 +1,91 @@
+{% load menu_tags cms_tags sekizai_tags staticfiles%}
```

Comment:
It works just as well outside the <html> tag, as it is in the base.html file under /ticket. Yeah should add <!DOCTYPE html>.